examples/plot_mandrill.py

```diff
@@ -0,0 +1,89 @@
+"""
+DiCoDiLe on the Mandrill image
+==============================
+
+This example illlustrates reconstruction of `Mandrill image
+<http://sipi.usc.edu/database/download.php?vol=misc&img=4.2.03>`_
+using DiCoDiLe algorithm with soft_lock "corner" and 16 workers.
+
+"""  # noqa
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+from dicodile.data.images import fetch_mandrill
+
+from dicodile.utils.dictionary import init_dictionary
+from dicodile.utils.viz import display_dictionaries
+from dicodile.utils.csc import reconstruct
+
+from dicodile import dicodile
+
+
+###############################################################################
+# We will first download the Mandril image.
+
+X = fetch_mandrill()
+
+plt.axis('off')
+plt.imshow(X.swapaxes(0, 2))
+
+
+###############################################################################
+# We will create a random dictionary of K = 25 patches of size 8 x 8 from the
+# original Mandrill image to be used for sparse coding.
+
+# set dictionary size
+n_atoms = 25
+
+# set individual atom (patch) size
+atom_support = (4, 4)
+
+# random state to seed the random number generator
+rng = np.random.RandomState(60)
+
+D_init = init_dictionary(X, n_atoms, atom_support, random_state=rng)
+
+###############################################################################
+# Set parameters.
+
+w_world = 3
+n_workers = w_world * w_world
```

Comment:
Maybe add a comment here saying that the computations are run with 9 workers on a 3x3 grid?